CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Introduce `helm-lint` which uses Helm chart testing to lint Helm Charts
```

Comment:
I don't consider commands to be real interface of the orb. Let's mention changes to the job instead which is important from the user perspective.

```suggestion
- Introduce Helm Chart testing and linting in `push-to-app-catalog` job.
```